infrastructure-examples/gcp_vm.yml

```diff
@@ -0,0 +1,28 @@
+cluster_name: gcloud-gke
```

Comment:
Change file name from `gcp_vm.yml` to avoid confusion with virtual machines.
`gcp_kubernetes.yml`, `gcloud_kubernetes.yml`, or other.